components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -394,11 +395,12 @@ interface EthTxController {
   // This returns different data depending on which network is currently selected in EthJsonRpcController
   GetAllTransactionInfo(string from) => (array<TransactionInfo> transaction_infos);
   AddObserver(pending_remote<EthTxControllerObserver> observer);
-  ApproveHardwareTransaction(string tx_meta_id) => (bool success, string message);
-  ProcessLedgerSignature(string tx_meta_id, string v, string r, string s) => (bool status);
-
   SpeedupOrCancelTransaction(string tx_meta_id, bool cancel) => (bool success, string tx_meta_id, string error_message);
   RetryTransaction(string tx_meta_id) => (bool success, string tx_meta_id, string error_message);
+  ApproveHardwareTransaction(string tx_meta_id) => (bool success);
+  GetTransactionInfo(string tx_meta_id) => (TransactionInfo? info);
+  GetTransactionMessageToSign(string tx_meta_id) => (string message);
+  ProcessHardwareSignature(string tx_meta_id, string v, string r, string s) => (bool status);
```

Comment:
please add a comment saying that v requires 0x and r, s don't require it.
Or just make them all need to have 0x for interface consistency. And it's better for Ledger to do `'0x' + v` instead of Trezor doing `replace('0x', '')`

Reply:
done